internal/server/authn/oidc.go

```diff
@@ -48,14 +47,14 @@ func (a *oidcAuthn) Authenticate(ctx context.Context, db *data.Transaction, requ
 		return AuthenticatedIdentity{}, fmt.Errorf("exhange code for tokens: %w", err)
 	}
 
-	if len(provider.AllowedDomains) > 0 {
+	if len(a.AllowedLoginDomains) > 0 {
 		// get the domain of the email
 		at := strings.LastIndex(idpAuth.Email, "@") // get the last @ since the email spec allows for multiple @s
```

Comment:
This seems to be the same logic as above `GetDomain`?

Reply:
This was a hold-over from an earlier implementation, removed it